python/housinginsights/ingestion/LoadData.py

```diff
@@ -312,6 +313,7 @@ def _create_zone_facts_table(self):
         # create empty zone_facts table
         metadata = MetaData(bind=self.engine)
         zone_facts = Table('zone_facts', metadata,
+                           Column('zone_type', String),
```

Comment:
In thinking through how this gets eventually used on the front end - I think we want to instead create the ZoneFacts table using meta.json. At a minimum, even if it is created using the sqlAlchemy table syntax , we need to have the duplicate information in meta.json because that is where we have the 'display_name' and 'display_text' for each of the fields, which (will be) used for things like tooltips on the front end.